tests/providers/microsoft/azure/sensors/test_wasb.py

```diff
@@ -59,6 +77,85 @@ def test_poke(self, mock_hook):
         mock_instance.check_for_blob.assert_called_once_with("container", "blob", timeout=2)
 
 
+class TestWasbBlobAsyncSensor:
```

Comment:
WDYT about adding a test that users verifies users can specify `timeout` and it's handled properly?